config.schema.yaml

```diff
@@ -14,3 +14,21 @@
     type: "string"
     secret: false
     required: true
+  sensor:
+    type: "object"
+    properties:
+      bitbucket_type:
+        type: "string"
+        description: "BitBucket service type"
+        required: true
+        enum:
+          - "cloud"
+          - "server"
+      bitbucket_server_url:
+        type: "string"
+        description: "URL of BitBucket server (this is only required to set in the case of the 'bitbucket_type' is set 'server')"
+        required: false
+      repositories:
+        type: "array"
```

Comment:
Should also add:

```yaml
items:
  type: "string"
```

This tells it that each array item needs to be a string.